packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -423,12 +425,9 @@ const SceneView = ({
           },
         }
       )}
-      freezeOnBlur={freezeOnBlur}
       // When ts-expect-error is added, it affects all the props below it
       // So we keep any props that need it at the end
       // Otherwise invalid props may not be caught by TypeScript
```

Comment:
Keep / remove this comment? 

I'm personally for keeping it in, so that we don't need to bring it back once new props emerge.